### PR TITLE
fix: Analytics scrolling

### PR DESCRIPTION
**Pull request by banklesss**
Screenshot 2023-08-11 at 14 09 23

Screenshot 2023-08-11 at 14 07 01

https://github.com/Emurgo/yoroi/assets/105349292/2adb0eed-5b34-4da6-83d0-25e97f3995e6

### CODE REVIEW

File: apps/wallet-mobile/src/components/Analytics/Analytics.tsx [hunk 1]
```diff
@@ -26,9 +26,17 @@ const Notice = ({onClose, onReadMore}: {onClose?: () => void; onReadMore?: () =>
   const strings = useStrings()
   const metrics = useMetrics()
 
+  const scrollViewRef = React.useRef<ScrollView | null>(null)
+
+  React.useEffect(() => {
+    setTimeout(function () {
+      scrollViewRef.current?.flashScrollIndicators()
+    }, 500)
```
Comment (michalszorad) [inline apps/wallet-mobile/src/components/Analytics/Analytics.tsx:38]: Can we also add cleanup? `return () => clearTimeout(timeout)`

**Review by michalszorad — COMMENTED** [review]
@banklesss good idea!

File: apps/wallet-mobile/src/components/Analytics/Analytics.tsx (hunk 1 above)
Comment (banklesss) [reply apps/wallet-mobile/src/components/Analytics/Analytics.tsx:38]: yeah!!

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/components/Analytics/Analytics.tsx [hunk 2]
```diff
@@ -22,13 +22,33 @@ export const Analytics = ({type, onClose, onReadMore}: Props) => {
   return <Notice onClose={onClose} onReadMore={onReadMore} />
 }
 
+// to display a border top in the button row
+const NOTICE_COMPONENT_HEIGHT = 700
```
Comment (michalszorad) [inline apps/wallet-mobile/src/components/Analytics/Analytics.tsx]: Is it possible to compute the `NOTICE_COMPONENT_HEIGHT` dynamically?

Comment (banklesss) [reply apps/wallet-mobile/src/components/Analytics/Analytics.tsx]: done

**Pull request merged**